trieste/acquisition/optimizer.py

```diff
@@ -83,7 +90,11 @@ def automatic_optimizer_selector(
 
     elif isinstance(space, (Box, TaggedProductSearchSpace)):
         num_samples = tf.minimum(NUM_SAMPLES_MIN, NUM_SAMPLES_DIM * tf.shape(space.lower)[-1])
```

Comment:
Unrelated to you changes, but it looks like there's something wrong here. It feels like this should either be:
```suggestion
        num_samples = tf.maximum(NUM_SAMPLES_MIN, NUM_SAMPLES_DIM * tf.shape(space.lower)[-1])
```
or
```suggestion
        num_samples = tf.minimum(NUM_SAMPLES_MAX, NUM_SAMPLES_DIM * tf.shape(space.lower)[-1])
```
???

Reply:
Good spot!

Reply:
good spot